lnwallet/channel_test.go

```diff
@@ -9092,437 +9092,6 @@ type heights struct {
 	remoteRemove uint64
 }
 
-func checkHeights(t *testing.T, update *paymentDescriptor, expected heights) {
-	updateHeights := heights{
-		localAdd:     update.addCommitHeights.Local,
-		localRemove:  update.removeCommitHeights.Local,
-		remoteAdd:    update.addCommitHeights.Remote,
-		remoteRemove: update.removeCommitHeights.Remote,
-	}
-
-	if !reflect.DeepEqual(updateHeights, expected) {
-		t.Fatalf("expected: %v, got: %v", expected, updateHeights)
-	}
-}
-
-// TestProcessAddRemoveEntry tests the updating of our and their balances when
-// we process adds, settles and fails. It also tests the mutating of add and
-// remove heights.
-func TestProcessAddRemoveEntry(t *testing.T) {
```

Comment:
CI failing rn, but I would like to sanity check coveralls to make sure we're not losing any coverage before merge.